lib/zoho_hub/base_record.rb

```diff
@@ -49,7 +49,7 @@ def where(params)
                      # see https://www.zoho.com/crm/help/developer/api/search-records.html
                      params
                    else
-                     { criteria: "#{attr_to_zoho_key(params.keys.first)}:equals:#{params.values.first}" }
+                     { criteria: "#{attr_to_zoho_key(params.keys.first)}:#{starts_with ? 'starts_with' : 'equals'}:#{params.values.first}" }
```

Comment:
I think that if you do this in the `else`, you won't be able to call `YourModule.start_with(phone: '01')`: the `email`, `phone` and `word` criteria won't work.

Reply:
From the [docs](https://www.zoho.com/crm/help/developer/api/search-records.html) it doesn't seem to me that `starts_with` can be passed to those three? Not sure haven't tried. The description on the table doesn't show that either, as well as the requested url. 🤔

Reply:
It does work also on these. E.g. if I have a `Zoho::Lead` class inheriting from `BaseRecord`, I can do this:
`Zoho::Lead.get("#{Zoho::Lead.request_path}/search", { criteria: "Email:starts_with:pkayo" })`
I correctyl retrieve the corresponding records.